components/request/request.html

```diff
@@ -1,36 +1,105 @@
-<div ng-controller="RequestController as model">
-  <div ng-show="model.loginRequired" ng-init="model.redirect()">
-    <h1 align="center">Log in</h1>
-    <form class="form-horizontal" role="form">
-      <fieldset>
-
-        <br-input br-model="model.username"
-          br-options="{name: 'username', label: 'Email'}">
-          Please enter your preferred email address.
-        </br-input>
-
-        <br-input br-model="model.password"
-          br-options="{name: 'password', label: 'Passphrase', type: 'password'}">
-          Please enter your passphrase (for example: 42RhinosStampedeAtTeaTime).
-        </br-input>
-
-        <div style="text-align:center;">
-          <input type="checkbox" name="public-terminal" ng-model="model.publicComputer"/> I'm using a public computer
+<div class="row" ng-controller="RequestController as model">
+  <div ng-show="model.display.identityChooser" ng-init="model.identityChooser()">
+    <h1 class="col-md-offset-3" style="padding-left: 40px">
+      <span ng-show="!model.identitySelected">Select an Identity</span>
+      <span ng-show="model.identitySelected">Log In</span>
+    </h1>
+    <div class="col-md-offset-3 col-md-6">
+      <div ng-hide="model.identitySelected && !identity.selected"
+        class="well" ng-repeat="identity in model.identities">
+        <div ng-show="identity.selected"
+          style="padding-bottom: 5px; border-color: #aaaaaa; border-bottom-style: solid; border-bottom-width: 1px; margin-bottom: 15px;">
+          <span ng-click="model.showAll(identity, loginForm);" style="cursor: pointer;">
+            <i class="fa fa-reply-all"></i><span style="padding-left: 5px">show all</span>
+          </span>
         </div>
-
-        <div style="text-align:center;">
-          <button ng-disabled="model.generating" ng-click="model.login(model.username,model.password)" type="submit" class="btn btn-primary">Login</button>
+        <div class="well" ng-class="{'br-selectable': !model.identitySelected}" ng-click="model.redirect(identity, loginForm)" style="cursor: pointer;">
+          <img src="http://simpleicon.com/wp-content/uploads/favorite-user-64x64.png">
+          <span style="margin-left: 30px"><strong>{{ identity.label }}</strong></span>
```

Comment:
By using bootstrap and font awesome, you probably don't have to worry about doing inline CSS style stuff like setting margins. Just use font awesome and choose the appropriate bootstrapism instead for combining an icon with something.